test/extended/testdata/ipsec/ipsec-nsconfig-machine-config.yaml

```diff
@@ -0,0 +1,54 @@
+# Generated by Butane; do not edit
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: 99-worker-north-south-ipsec-config
+spec:
+  config:
+    ignition:
+      version: 3.4.0
+    storage:
+      files:
+        - contents:
+            compression: gzip
+            source: data:;base64,H4sIAAAAAAAC/2TTOdOyTBYG4Jxf0bk1hbLKV/UETXeD7LSgiJkgsgrKqvz6qeedqgne70TnXMG5o1u95QCOY18m05gNDAAAPPoya+/N1709s38Agn+w6dJbY2VfA/8DRAUIAhBFoGhAJUDigLYF3BbIKtjugCIDEQFVBgoP9grQtN8FawApQOUAM0xJlaXjD3LBz+/zchimrP//+Z/fUYluuACRY2hoBoIh+aOMYxioWRGCl00OF0OFuaFBxxLLsl4wjU2ruxrFnLqQElulcMEhsR1Y63B3IvDjIIZuyXIoUtep6OJi8nFwvDqV8Yl+bf3LKug6dFgQjfGZUkwWEzM0RAgGxt9pqkohznPiQ4wQpB3Kc6JCWz5zgmLf9i/zZUu9Ng0e06W2jJ7U4NpqUxiPoHDVbjQ/Se5KL9aRlOBqzKckjyof8oeR3wgTOh7Ghxc/ycMd5Iw5wjBEKi73nmDC9CzOUSsUrz5R+gHziNWFaVsd+ZY97d96M73O3e7eyl894D/ivRW9lCFpI30FKWlziK0wtYU21RaDjaqUYvKYVmcO0Tftj81lz40J6zR2pMSD5WuG4M3CujCzOISXPI3Z72zvheHw7c+rv1HLUtE5rEfXemxxnZnBWKuFhOvn/nWBukj8HV2LOdBEZoo6ao8Unh7Xfgiu7ChNvugX8BJbu+PY4DLlC6c7VNe1ImMhitmtrHYX61wcxs3FMQhjrQTBhUAYQlfN63dRl7qybFVIBw1CD0FKYMCj4xi3hkJumlEM92yVbo9oTCr7WzLyK6yXB9tMLeeYmhn1yvzOY1UictwEbOFm3EtfuYYGdf/2Oy5v5G7SI0TWyOTuj55jnMSnKxt61uFITv3He505m8tR+j2929MpmqdFi6T3hEzp6UWpLV3fmRWd09o6fE/2/LkxfutFz85Udp4rjnKg17BLNnu+osfH54bM/BbuNPwR0rYPhvtVKfY6ir3vfSuYlygRWZcpnVnF7HojgpK/YT6J6vaYi893jJ3pPuxEEvd1mYutdq2uWrb3T5cmmj2MVDnlrF09M/qinHZKmMtbPbLmbCvZFuvcEZsi1xB4Pn/Rt32enIH3OaM1NzTdmElBdlqx/Pz8r6XExf/u6H8DAAD//zcw/5BQBAAA
```

Comment:
curious how the name on the certificates ended up not being a problem?

Reply:
yes, we use the same certificate name in NNCP while configuring libreswan on the node.
https://github.com/openshift/origin/pull/28658/files#diff-171496901153a36ea9127c00c510f03758d598927be689320eaeb044f464e932R77-R80

Reply:
Can you elaborate? That link is not taking me anywhere.

Reply:
ok, i meant about these are the cert names https://github.com/pperiyasamy/origin/blob/ipsec-e2e-tests/test/extended/networking/ipsec.go#L77-L80 being used in `nodeIPsecConfigManifest`

Reply:
ah! ok, this is the script being used to generate left and right certs.
https://github.com/openshift/origin/blob/e7382d905fac7db8e2994b3f530d8703bd61e338/test/extended/networking/ipsec.go#L77-L91
common name is hard coded to left and right servers and for `extSAN`, ip is replaced with email, hence there is no problem with using offline generated certs. Will update e7382d905fac7db8e2994b3f530d8703bd61e338 to reflect email on the `extSAN` parameter.

Reply:
Oh. So nothing is actually trying to match those names with the real hostnames? That's...unexpected.

Reply:
correct, but after importing this certificate into nss db, this is being referred from NNCP created for the node.
https://github.com/pperiyasamy/origin/blob/ipsec-e2e-tests/test/extended/networking/ipsec.go#L63